docs/hlo_passes.md

```diff
@@ -0,0 +1,165 @@
+# HLO Passes 
```

Comment:
For dev details on using passes, consider linking to:

https://github.com/openxla/xla/blob/2f445596f8d6d6751b27c4c976df0cc0cb1f9631/xla/hlo/pass/README.md?plain=1#L4